tests/conftest.py

```diff
@@ -120,7 +120,8 @@ def notify_db_session(notify_db, sms_providers):
                             "broadcast_status_type",
                             "invite_status_type",
                             "service_callback_type",
-                            "broadcast_channel_types"]:
+                            "broadcast_channel_types",
```

Comment:
Ahh, is it where we clean test db when running tests?

Reply:
Yes, and the tables in this list are the ones that we don't want to clean since they contain fixed data that should always exist.

Reply:
makes sense, thanks! ☺️